deploy-to-ghpages.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+function doEverythingElse() {
+  commit="Deploying master to gh-pages by $USER"
+  echo If you would like to include a commit message, add one and press RETURN.
+  read personalized
+  if [ -z "$personalized" ]; then
+    $commit = "$commit: $personalized"
+  fi
+  git commit -m "$commit"
+  echo Done committing.
+
+  echo Ready to push? Caution: your changes will be live on the internet! [y/n]
+  read confirm
+  if [ "$confirm" = "y" ]; then
+    git push origin gh-pages
+    echo Done pushing.
+  else
+    echo Oh well. Run this script again later when ready.
+  fi
+  cd ../../
+  echo --------------------
+  echo Deleting the temporary repo copy.
+  rm -rf multiplatform-deploy-tmp
+  echo Done deleting.
+  cd pvta-multiplatform
+}
+
+cd ../
```

Comment:
Note that this script *could* be executed from not the current directory perhaps try `dirname $0`